<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotoAddWord</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lab_display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string>显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_input">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>导入按键</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_preview">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>451</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string>预览图片</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
